custom/templates/DefaultRevamp/authme_email.tpl

```diff
@@ -31,17 +31,64 @@
                     </div>
                 </div>
                 <form class="ui form" action="" method="post" id="form-authme-email">
-                    <div class="field">
-                        <label for="inputEmail">{$EMAIL}</label>
-                        <input type="email" id="inputEmail" name="email" placeholder="{$EMAIL}">
-                    </div>
+                    {assign var=counter value=1}
```

Comment:
We should look into reusable tpl files 🤔 😅

Reply:
make the fields stuff into a include tpl? Yea i thinked about doing that when i made it, same as few of my modules will also use this